lib/Resque.php

```diff
@@ -337,4 +337,38 @@ public static function generateJobId()
         return md5(uniqid('', true));
     }
 
+    public static function getInProgressJobsCount() {
+        return self::redis()->hlen('current_jobs');
+    }
+
+    /**
+     * Clean workers that were terminated after timeout
+     *
+     * @return array of unfinished jobs
+     */
+    public static function cleanWorkers() {
+        $notFinishedJobs = [];
+        $workers = self::redis()->sMembers('workers');
```

Comment:
is there a separation between prod and prod-rc?

Reply:
Added prefix parameter to separate between prod and prod-rc